statics/js/components/preference.js

```diff
@@ -0,0 +1,132 @@
+/* jshint multistr: true */
+
+var PreferenceComponent = {
+
+  name: 'Preference',
+
+  mixins: [notificationMixin],
+
+  template: '\
+    <form class="form-preference" @submit.prevent="save">\
+      <div class="form-group">\
+        <label for="default-filter">Filters</label>\
+        <a><i class="fa fa-question help-text" aria-hidden="true" title="filter and highlight gremlin queries, displayed in the left panel"></i></a>\
+        <div v-for="filter in filters">\
+          <div class="form-group">\
+            <div class="input-group">\
+              <label class="input-group-addon">Name: </label>\
+              <input class="form-control" v-model="filter.name"/>\
+            </div>\
+            <div class="input-group filter-field">\
+              <label class="input-group-addon">Filter:  </label>\
+              <input class="form-control" v-model="filter.filter"/>\
+            </div>\
+          </div>\
+        </div>\
+        <button class="btn btn-primary btn-round-xs btn-xs" type="button" @click="addFilter">+</button>\
+      </div>\
+      <div class="form-group">\
+        <label for="bw-threshold">Bandwidth Threshold</label>\
+        <a><i class="fa fa-question help-text" aria-hidden="true" title="bandwidth threshold mode (relative/absolute)"></i></a>\
+        <select id="be-threshold" v-model="bwThreshold" class="form-control input-sm">\
+          <option value="">None</option>\
```

Comment:
This can't be None it's either Absolute or Relative the default is Relative, but we should use the value coming from the config as default. If the storage is not filled with fill the Panel with the values from the API

Reply:
done. now if there is no value in local storage, it will take from the config file.